app/grandchallenge/components/templates/components/partials/ci_example_value.html

```diff
@@ -0,0 +1,36 @@
+{% load json%}
+
+<a href="#ciModal-{{ interface.pk }}"
+    data-toggle="modal"
+    data-target="#ciModal-{{ interface.pk }}"
+    class="text-decoration-none"
+    role="button"
+    title="View {{ interface.title }}"
+>
+    <div class="badge badge-primary">
+        <i
+                class="fas fa-fw fa-eye"
+        ></i>
```

Comment:
What is all this whitespace?

```suggestion
        <i class="fas fa-fw fa-eye"></i>
```

Reply:
This is strange. It happened a couple of times before. I did not write it like this. I suspect something wrong is happening on pre-commit. Is there a specific config should be set there? I will check again the changes after pre-commit before pushing.

Reply:
Pre commit would never do this.

Reply:
oh, sorry. I just realized the original template `modal_base.html` where I got the code already has it like this. I thought you already approved the available code and that is why I try to use as much as possible. I will check more carefully next time.

Reply:
I am having to go through your code much more closely as what you give me to review has a lot of problems including breaking existing functionality that I had to revert (ab18aed), work where you didn't address PR feedback and I have to step in to complete it (#3462), and work that I had to re-do due to an overly-complex implementation (#3464). Please critically read the code you request review for before asking me.

Reply:
Done